templates/live.html

```diff
@@ -0,0 +1,52 @@
+{% extends "tabulator.html" %}
+{% block active_tab %}0{% endblock %}
+{% block tab_head %}
+<script src="../static/js/tab1.js" type="module"> </script>
+{% endblock %}
+{% block tab_content %}
+<noscript>
+    <h2>Liveübersicht</h2>
+   <h3>Leider kann hier ohne JavaScript nichts angezeigt werden. Die anderen Tabs funktionieren aber!</h3>
+</noscript>
+<div class="js_enabled" style="display: none;" data-tab="0">
```

Comment:
Is the inline style necessary here? If not, please use the tailwind `hidden`-class; else please add a comment clarifying the requirement of the inline style.